Doc/library/asyncio-eventloop.rst

```diff
@@ -1641,6 +1641,25 @@ Do not instantiate the :class:`Server` class directly.
       coroutine to wait until the server is closed (and no more
       connections are active).
 
+   .. method:: close_clients()
+
+      Close all existing incoming client connections.
```

Comment:
Having now read the code, I wonder if this deserves an example or a reminder that the proper protocol is to call `close()` first, `close_clients()` second? Because if you call the latter first, there's a race condition where new connections could be made before the socket is closed.

Reply:
Indeed. So that might be worth a note.